src/test/java/com/evanlennick/retry4j/RetryConfigBuilderTest_ValidationDisabled.java

```diff
@@ -73,7 +73,7 @@ public void testSettingDurationBetweenTries_seconds() {
     @Test
     public void testSettingDurationBetweenTries_millis() {
         RetryConfig config = retryConfigBuilder
-                .withDelayBetweenTries(5000L)
+                .withDelayBetweenTries(5, ChronoUnit.MILLIS)
                 .build();
 
         assertThat(config.getDelayBetweenRetries().toMillis()).isEqualTo(5000);
```

Comment:
@dmcneil probably dont need both tests anymore but it doesnt really hurt having them